ci/jenkins/test.sh

```diff
@@ -896,7 +898,7 @@ function redeploy_k8s_if_ip_mode_changes() {
         else
             NODE_IP_STRING="${IPV4S[i]},${IPV6S[i]}"
         fi
-        ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/id_rsa" -n ubuntu@${IPV4S[i]} "echo \"KUBELET_EXTRA_ARGS=--node-ip=${NODE_IP_STRING}\" | sudo tee /etc/default/kubelet; sudo systemctl restart kubelet"
+        ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/id_rsa" -n ubuntu@${IPV4S[i]} "echo \"KUBELET_EXTRA_ARGS=--node-ip=${NODE_IP_STRING}\" | sudo tee /etc/default/kubelet; sudo systemctl unmask kubelet; sudo systemctl restart kubelet"
```

Comment:
I don't find any code masking the service, why it needs to be unmasked?

Reply:
After we installed kubelet 1.28.9, the service is in masked state, we need to unmask it first.

Reply:
Is this service always masked in newer K8s?

Reply:
The service may be masked for the K8s package in the new repo.
We changed the apt repo in testbed VM since old K8s repo is unavailable:
`https://apt.kubernetes.io/`
->
`https://pkgs.k8s.io/core:/stable:/v1.30/deb/`

Reply:
Can we have the K8s apt repository update in this script? If test.sh will also re-install K8s in IPAM testbed, I feel it's unnecessary to require users to update the `https://pkgs.k8s.io/core:/stable:/v1.30/deb` to a newer one manually on testbeds in the future.